### PR TITLE
Populate Users 

### PR DESCRIPTION
**Pull request by corybrooks**
This PR implements the feature to populate the current users and display their usernames in the Users fragment of the app.

### CODE REVIEW

**Review by ymittal — APPROVED** [review]
LGTM. Just remove extra line spacings in some files.

File: android/app/src/main/java/com/google/codeu/chatme/view/adapter/UserListAdapter.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    public void loadUsers() {
+        this.presenter.loadUsers();
+    }
+
+
+    @Override
+    public void onBindViewHolder(UserListAdapter.ViewHolder holder, int position) {
+        final User user = users.get(position);
+        holder.tvName.setText(user.getName());
```
Comment (blong1996) [inline android/app/src/main/java/com/google/codeu/chatme/view/adapter/UserListAdapter.java]: name has been changed to`fullName` and `username` in the  `User` model. merge the latest version of `dev` to this branch and make the necessary changes. I would suggest attempting to grab each user's Full Name first and, if empty, grab their username instead.

**Review by blong1996 — CHANGES_REQUESTED** [review]
pull latest changes from dev  branch and update references made to User objects in UserListAdapter, line 46

**Review by ymittal — APPROVED** [review]
LGTM.

**Review by blong1996 — APPROVED** [review]

**Pull request merged**